lib/openstudio-standards/btap/structure.rb

```diff
@@ -351,6 +360,23 @@ def initialize(model = nil, cat = "commerce")
         end
       end
 
+      # Reset :clt and :metal structure selections - not yet available. @todo
+      @structure = :steel    if @structure == :metal
+      @structure = :concrete if @structure == :clt
```

Comment:
Auto-switching _CLT_ (mass timber) and _metal_ buildings to _concrete_ post/flat slab and _steel_ post/beam structures, respectively.